sql/src/main/java/io/crate/executor/transport/AlterTableOperation.java

```diff
@@ -86,18 +88,24 @@
     private final TransportActionProvider transportActionProvider;
     private final Provider<TransportOpenIndexAction> transportOpenIndexActionProvider;
     private final Provider<TransportCloseIndexAction> transportCloseIndexActionProvider;
+    private final TransportRenameTableAction transportRenameTableAction;
+    private final TransportIndicesAliasesAction transportIndicesAliasesAction;
     private final SQLOperations sqlOperations;
 
     @Inject
     public AlterTableOperation(ClusterService clusterService,
                                TransportActionProvider transportActionProvider,
                                Provider<TransportOpenIndexAction> transportOpenIndexActionProvider,
                                Provider<TransportCloseIndexAction> transportCloseIndexActionProvider,
+                               TransportRenameTableAction transportRenameTableAction,
```

Comment:
Is there any reason we are using providers for the open/close actions, but not for the rename/aliases? If no, it'd be cool if we unified them and either made them all use providers or all use actions